charts/cluster-api-runtime-extensions-nutanix/addons/nfd/values-template.yaml

```
@@ -1,3 +1,5 @@
+priorityClass: system-node-critical
```

```suggestion
priorityClassName: system-node-critical
```
https://kubernetes-sigs.github.io/node-feature-discovery/v0.16/deployment/helm.html
we also need to set
```
worker:
  priorityClassName: system-node-critical
```

---

Thanks for being vigilant. However, having looked at the Chart, the top level `priorityClassName` field value is propagated to all components--master, workers, and gc :smile:

---

:) . Thanks for checking the code. I will file bug/fix upstream. 
We can just change top level `priorityClassName` then.

---

You think there's a bug to report?

It seems acceptable to have both the top-level field and lower-level `priorityClassName` fields in the Helm chart values. :shrug: